TESTS/netsocket/udp/main.cpp

```diff
@@ -180,10 +191,14 @@ Case cases[] = {
     Case("UDPSOCKET_BIND_UNOPENED", UDPSOCKET_BIND_UNOPENED),
     Case("UDPSOCKET_SENDTO_INVALID", UDPSOCKET_SENDTO_INVALID),
     Case("UDPSOCKET_ECHOTEST_NONBLOCK", UDPSOCKET_ECHOTEST_NONBLOCK),
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE != MESH
     Case("UDPSOCKET_ECHOTEST_BURST_NONBLOCK", UDPSOCKET_ECHOTEST_BURST_NONBLOCK),
+#endif
```

Comment:
This is something we should not skip.
Its basically close to what DTLS handshake is sending. Not accurate, but still emulates it.
Now we are just hiding problems.

Reply:
Mostly up 90% of the  UDP bursts fails
This test also must be run  under "no RF traffic "and "clear channel" condition.

Reply:
Test skip reverted. It is successful under  local mesh BR. Testing mesh performance under heavy RF traffic is beyond the scope of netsockets tests.

Reply:
Test skip reverted. It is successful under  local mesh BR. Testing mesh performance under heavy RF traffic is beyond the scope of netsockets tests.

Reply:
Test skip reverted. It is successful under  local mesh BR. Testing mesh performance under heavy RF traffic is beyond the scope of netsockets tests.